app/views/pdfjs_viewer/viewer/_viewer.html.erb

```diff
@@ -23,14 +22,14 @@ Adobe CMap resources are covered by their own copyright but the same license:
 
 See https://github.com/adobe-type-tools/cmap-resources
 -->
-<html dir="ltr" mozdisallowselectionprint moznomarginboxes>
+<html dir="ltr" mozdisallowselectionprint>
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <meta name="google" content="notranslate">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <%= render "pdfjs_viewer/viewer/head", title: title, pdf_url: pdf_url %>
-    <%= render "pdfjs_viewer/viewer/extra_head", local_assigns %>
```

Comment:
May I ask why you deleted this? 😄

Reply:
I don’t remember to be honest :(. I would guess that I only removed it because it caused some issue but might not be the case.

Reply:
[UDPATE](https://github.com/senny/pdfjs_viewer-rails/blob/master/UPDATE.md#appviewspdfjs_viewerviewer_viewerhtmlerb) says so. I think @gingermusketeer did it. The update.md should also be updated.